### ISSUE TITLE
bug: WCAG Pointer Cancellation (2.5.2) is no accomplished on iOS

### ISSUE THREAD

**Issue by crazyserver** [ionic-team/cordova-plugin-ionic-webview#644]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**

Pointer cancellation criterion is not accomplished on Ionic main components, while a click is released out of the button target space the click action is triggered.

[Screen recording (aspect ratio lost during video conversion)](https://pasteboard.co/K5Wskpd.gif)

**Expected behavior:**

The click action should be cancelled and not triggered giving the user the possibility to cancel the tap.

**Steps to reproduce:**

1. Create a new "tabs" app and run it.
2. Tab onto a tab but release the finger out of the tab bar.
3. Error behaviour: the tab changed.
4. Successful behaviour: the click is not triggered and the tab is not selected.

**Related code:**

```
ionic start myApp tabs --cordova
cd myApp
ionic cordova platform add ios
ionic cordova build ios
```

**Other information:**

- [WCAG 2.5.2 Pointer Cancellation information](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html)

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.14.1 (/Users/dpalou/.nvm/versions/node/v14.15.0/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.6.9
   @angular-devkit/build-angular : 12.0.3
   @angular-devkit/schematics    : 12.0.3
   @angular/cli                  : 12.0.3
   @ionic/angular-toolkit        : 4.0.0

Cordova:

   Cordova CLI       : 10.0.0
   Cordova Platforms : ios 6.2.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 4 other plugins)

Utility:

   cordova-res                          : not installed
   native-run (update available: 1.4.0) : 1.3.0

System:

   Android SDK Tools : 26.1.1 (/Users/dpalou/Library/Android/sdk)
   ios-deploy        : 1.11.4
   ios-sim           : 8.0.2
   NodeJS            : v14.15.0 (/Users/dpalou/.nvm/versions/node/v14.15.0/bin/node)
   npm               : 6.14.8
   OS                : macOS Big Sur
   Xcode             : Xcode 12.5 Build version 12E262
```

**Comment by crazyserver** [comment]
Thanks @liamdebeasi !
